benchmarks/user/README.md

```diff
@@ -211,6 +211,51 @@ Do not forget to pin processes, especially on NUMA systems; below, we do so with
 ```
 numactl --cpubind=0 --membind=0 python benchmark.py ...
 ```
+
+## Benchmarking performance
+
+In order for a user to benchmark the performance of an operator calculating
+GFlops/s, Operational Intensity and Execution timings the `bench` mode should
+be used while also providing as an environment variable `DEVITO_LOGGING=DEBUG`.
+
+So the isotropic acoustic wave forward Operator in a `512**3` grid, space order
+12, and a simulation time of 100ms:
+
+```
+`DEVITO_LOGGING=DEBUG` python benchmark.py bench -P acoustic -d 512 512 512 -so 12 --tn 100
+```
+
+A `.json` file will be created in a folder named `results` except if otherwise specified
+with the `-r` option specifying the results directory.
+
+
+## Plotting a roofline
+
+In order for a user to plot the results obtained from the `bench` in the previous
+paragraph user shoud execute `benchmark.py` in plot mode. It is essential that 
+some more arguments should be provided at this stage.
+The necessary arguments can be also seen through `python benchmark.py plot --help`.
+
+For correct results, user should carefully specify as additional arguments:
+
+* --max-bw FLOAT that is          Max GB/s of the DRAM
+* --flop-ceil <FLOAT TEXT>...     Max GFLOPS/s of the CPU. A 2-tuple (float, str)
+is expected, where the float is the performance ceil (GFLOPS/s) and the str
+indicates how the ceil was obtained (ideal peak, linpack, ...)
+* --point-runtime                 Annotate points with runtime values
+
+An example for the benchmarked problem mentioned in previous paragraph
+on a system with 12.8 GB/s of the DDRAM and 80 GFLOPS/s using Linpack as the
+performance ceiling would be:
+ 
+```
+python benchmark.py plot -P acoustic -d 512 512 512 -so 12 --tn 100 --max-bw 12.8
+ --flop-ceil 80 "linpack"
+```
+
+In order to derive --max-bw for your machine we strongly advise that you use
+[STREAM][http://www.cs.virginia.edu/stream/ref.html].
+
 While a benchmark is running, you can have some useful programs running in
```

Comment:
where should this now go?

Reply:
Probably below "Example commands" 

You mistakenly pushed it down inside another section